src/js/docs.js

```diff
@@ -1 +1,31 @@
 import '@github/clipboard-copy-element'
+import { on } from 'delegated-events'
+
+on('click', '[data-copy-feedback]', ({ currentTarget }) => {
+  const text = currentTarget.getAttribute('data-copy-feedback')
+  const el = createBubble(text, { fade: true })
+  currentTarget.appendChild(el)
+})
+
+function createBubble (text, options) {
+  const bubble = document.createElement('div')
+  bubble.classList.add(
+    'rounded-4', 'px-8', 'py-4',
+    'bg-slate-4', 'text-white', 'text-small',
+    'absolute', 'top-full', 'right-0', 'mt-4'
+  )
+  bubble.textContent = text
+  if (options?.fade) {
```

Comment:
this seems like a good candidate for christina's eyes. no need to hold the PR up but wanted to plant the seed